### PR TITLE
Jenkins 2.89.2 LTS + plugins

### PR DESCRIPTION
**Pull request by falseperfection**
universe/repo/packages/J/jenkins$ diff -ur 23 24

```diff
diff -ur 23/marathon.json.mustache 24/marathon.json.mustache
--- 23/marathon.json.mustache	2017-12-13 12:02:29.000000000 -0600
+++ 24/marathon.json.mustache	2017-12-14 11:59:55.000000000 -0600
@@ -40,7 +40,7 @@
            "image": "{{advanced.docker-image}}",
        {{/advanced.docker-image}}
        {{^advanced.docker-image}}
-           "image": "{{resource.assets.container.docker.jenkins-330-2731}}",
+           "image": "{{resource.assets.container.docker.jenkins-340-2892}}",
        {{/advanced.docker-image}}
            "network" : "HOST"
        },
diff -ur 23/package.json 24/package.json
--- 23/package.json	2017-12-13 12:02:29.000000000 -0600
+++ 24/package.json	2017-12-14 12:00:00.000000000 -0600
@@ -1,7 +1,7 @@
 {
   "packagingVersion": "3.0",
   "name": "jenkins",
-  "version": "3.3.0-2.73.1",
+  "version": "3.4.0-2.89.2",
   "minDcosReleaseVersion": "1.8",
   "scm": "https://github.com/mesosphere/dcos-jenkins-service.git",
   "maintainer": "support@mesosphere.io",
diff -ur 23/resource.json 24/resource.json
--- 23/resource.json	2017-12-13 12:02:29.000000000 -0600
+++ 24/resource.json	2017-12-14 12:31:38.000000000 -0600
@@ -7,7 +7,7 @@
   "assets": {
     "container": {
       "docker": {
-        "jenkins-330-2731": "mesosphere/jenkins:3.3.0-2.73.1"
+        "jenkins-340-2892": "mesosphere/jenkins:3.4.0-2.89.2"
       }
     }
   }
```

### CODE REVIEW

File: repo/packages/J/jenkins/24/resource.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-small.png",
+    "icon-medium": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-medium.png",
+    "icon-large": "https://downloads.mesosphere.com/universe/assets/icon-service-jenkins-large.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "jenkins-340-2891": "mesosphere/jenkins:3.4.0-2.89.2"
```
Comment (colin-msphere) [inline repo/packages/J/jenkins/24/resource.json]: This field name could be updated to match the image name.

File: repo/packages/J/jenkins/24/marathon.json.mustache [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      {"port": 0, "protocol": "tcp", "name": "jenkins"}
+  ],
+  "container": {
+       "type": "DOCKER",
+       "docker": {
+       {{#advanced.docker-image}}
+           "image": "{{advanced.docker-image}}",
+       {{/advanced.docker-image}}
+       {{^advanced.docker-image}}
+           "image": "{{resource.assets.container.docker.jenkins-340-2892}}",
```
Comment (colin-msphere) [inline repo/packages/J/jenkins/24/marathon.json.mustache]: This resource does not match what is in `resource.json`.

File: repo/packages/J/jenkins/24/resource.json (hunk 1 above)
Comment (falseperfection) [reply repo/packages/J/jenkins/24/resource.json]: Argh! I missed one!

**Comment by colin-msphere** [comment]
Tested, deployed, and ran jobs as expected.

**Pull request merged**